src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/FileUploader.vue

```diff
@@ -208,7 +231,7 @@ export default defineComponent({
         }
       } catch (error) {
         this.removeFile(selectedFile);
-        this.alertStore.setFailureAlert("An error occurred during file upload");
+        this.alertStore.setFailureAlert("Problem with your internet connection. Check your connection and try again.");
```

Comment:
I know this is in the acceptance criteria ... but are we certain that the user is having internet connection issues? 

I think this can happen if ClamAV is down as well. 

Message seems misleading that's all. Could just tell the user to try again.

Reply:
Resolved